### ISSUE TITLE
Narrator gives inappropriate announcement on selecting text in 'Search' edit field.

### ISSUE THREAD

**Issue by abbasms** [microsoft/microsoft-ui-xaml#1966]
**Describe the bug**
When selecting text content in the Search box (in the navigation menu), the Narrator announces 'A document selected', instead of the text.

**Steps to reproduce the bug**

- Open the Xaml Controls Gallery app and the narrator.
- Type something in the Search box.
- Select the text in the Search box (Ctrl + A). (Consistently reproducible if the narrator focus is on a single letter.)
- See that narrator does not announce the selected text, and instead reads out 'A document selected'.

**Expected behavior**
Selected text is read out by the narrator.

**Version Info**
NuGet package version:
2.3.191211002 and 2.2.190917002

Windows 10 version | Saw the problem?
-- | --
Insider Build (xxxxx) |  
November 2019 Update (18363) | Yes
May 2019 Update (18362) |  
October 2018 Update (17763) |  
April 2018 Update (17134) |  
Fall Creators Update (16299) |  
Creators Update (15063) |  

Device form factor | Saw the problem?
-- | --
Desktop | Yes
Mobile |  
Xbox |  
Surface Hub |  
IoT

**Comment by YuliKl** [comment]
Based on internal discussion with Narrator team, this behavior is by design.
